Sources/Redbird/Parsers.swift

```diff
@@ -177,9 +177,7 @@ struct ArrayParser: Parser {
         }
         let rawCountString = try head.stringView()
         let countString = rawCountString.strippedInitialSignatureAndTrailingTerminator()
-        guard let count = Int(countString) else {
-            throw RedbirdError.arrayProvidedUnparseableCount(countString)
-        }
+        let count = Int(countString) ?? 3 // Default to 3 if not specified
```

Comment:
@cwoloszynski can we add a note here as to why 3 or more preferable, a named variable like `let whatThreeMeans = 3`